src/Microsoft.AspNetCore.Razor.Test.Sources/CaseSensitiveTagHelperAttributeComparer.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Razor.Test.Internal
```

Comment:
Yup. Note, I'd do a search of all the repos for the name because i believe we have special entries for it within our build system 😄 .